### DISCUSSION TITLE
Welcome to FinBooks Discussions!

### DISCUSSION THREAD

**Discussion by Kanav-Arora** [Kanav-Arora/FinBooks#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ya-s-h** [comment]
Hello Kanav, It would be great if you can either host your project somewhere so that others can try this project, or add instructions for others to host it by themselves on their local machine.

**Reply by Kanav-Arora** [reply]
Hey @Ya-s-h,
I have just launched a teaser for the product. The documentation is under process. I'm planning to launch it on mobile platforms by next month. I hope you'll wait with us. 
This project is solely created and managed by me, therefore, all processes are a bit slow. I hope you understand.
If you want to stay updated with news regarding FinBooks and my other products, you can connect with me on [LinkedIn](https://www.linkedin.com/in/kanavarora1505/). I regularly post updates there.
Regards,
Kanav Arora
